lib.sh

```diff
@@ -675,6 +682,30 @@ exit
 fi
 }
 
+set_max_count() {
+if grep -F 'vm.max_map_count=262144' /etc/sysctl.conf ; then
+	echo "Max map count already set, skipping..."
+else
+	sysctl -w vm.max_map_count=262144
+	echo "#Docker ES max virtual memory
+	vm.max_map_count=262144" >> /etc/sysctl.conf
+fi
+}
+
+install_docker() {
+if [ $DOCKER_INS == "docker-ce" ] || \
```

Comment:
We have a function for that: `is_this_installed [package_name]`

Reply:
But its purpose is not the same, here we only look if the package is installed to prevent run the docker installation script.
* docker.io -> v17 
* docker-ce/ee -> v18

All of them are supported versions.
`is_this_installed` will break the process.

Reply:
OK, fine by me then.